build-tag-release/action.yml

```
@@ -69,6 +77,22 @@ runs:
         [[ "$VERSION" != "" ]] || exit 1
         echo "VERSION=$VERSION" >> $GITHUB_ENV
 
+    - name: Add prerelease details
+      if: ${{ inputs.prerelease == 'true' }}
+      shell: bash
+      run: |
+        if [[ -z "${{ inputs.main_plugin_file }}" ]]; then
+          echo "No main plugin file specified for prerelease edits."
+          exit 0
```

i would probably `exit 1` here. If you screwed up and forgot to add the plugin file name, with no default, you need to fix your workflow.

---

I went back and forth on that. good call.
```suggestion
          exit 1
```